runway/blueprints/staticsite/auth_at_edge.py

```diff
@@ -0,0 +1,390 @@
+            'CacheBehaviors': [
+                cloudfront.CacheBehavior(
+                    PathPattern=variables['RedirectPathSignIn'],
+                    Compress=True,
+                    ForwardedValues=cloudfront.ForwardedValues(
+                        QueryString=True
+                    ),
+                    LambdaFunctionAssociations=[
+                        cloudfront.LambdaFunctionAssociation(
+                            EventType='viewer-request',
+                            LambdaFunctionARN=parse_auth_lambda_version.ref()
+                        )
+                    ],
+                    TargetOriginId='dummy-origin',
+                    ViewerProtocolPolicy="redirect-to-https"
+                ),
+                cloudfront.CacheBehavior(
+                    PathPattern=variables['RedirectPathAuthRefresh'],
+                    Compress=True,
+                    ForwardedValues=cloudfront.ForwardedValues(
+                        QueryString=True
+                    ),
+                    LambdaFunctionAssociations=[
+                        cloudfront.LambdaFunctionAssociation(
+                            EventType='viewer-request',
+                            LambdaFunctionARN=refresh_auth_lambda_version.ref()
+                        )
+                    ],
+                    TargetOriginId='dummy-origin',
+                    ViewerProtocolPolicy="redirect-to-https"
+                ),
+                cloudfront.CacheBehavior(
+                    PathPattern=variables['SignOutUrl'],
+                    Compress=True,
+                    ForwardedValues=cloudfront.ForwardedValues(
+                        QueryString=True
+                    ),
+                    LambdaFunctionAssociations=[
+                        cloudfront.LambdaFunctionAssociation(
+                            EventType='viewer-request',
+                            LambdaFunctionARN=sign_out_lambda_version.ref()
+                        )
+                    ],
+                    TargetOriginId='dummy-origin',
+                    ViewerProtocolPolicy="redirect-to-https"
+                ),
+            ],
+            'DefaultCacheBehavior': cloudfront.DefaultCacheBehavior(
+                AllowedMethods=['GET', 'HEAD'],
+                Compress=True,
+                DefaultTTL='86400',
+                ForwardedValues=cloudfront.ForwardedValues(
+                    QueryString=True,
+                ),
+                LambdaFunctionAssociations=default_cache_behavior_lambdas,
+                TargetOriginId='protected-bucket',
+                ViewerProtocolPolicy='redirect-to-https'
+            ),
+            'DefaultRootObject': 'index.html',
+            'Logging': self.add_logging_bucket(),
+            'PriceClass': variables['PriceClass'],
+            'Enabled': True,
+            'WebACLId': self.add_web_acl(),
+            'CustomErrorResponses': [
+                cloudfront.CustomErrorResponse(
```

Comment:
The 403 error response should be dropped (incorporate https://github.com/aws-samples/cloudfront-authorization-at-edge/pull/26)